plantcv/plantcv/hyperspectral/rot90.py

```diff
@@ -0,0 +1,44 @@
+# Rotate hyperspectral datacubes (in increments of 90 degrees)
```

Comment:
Would add "clockwise", i.e. Rotate hyperspectral datacubes counterclockwise (in ...

Reply:
Great point!